src/background/IndexedDBDataKeys.ts

```diff
@@ -3,4 +3,5 @@
 export class IndexedDBDataKeys {
     public static readonly assessmentStore: string = 'assessmentStoreData';
     public static readonly userConfiguration: string = 'userConfiguration';
+    public static readonly installation: string = 'installation';
```

Comment:
is possible to make the value `'installationData'` instead? I'm thinking about the code we found that may be able to generate this keys from, let's say, an associated store. Ensure that we follow the same pattern on the string may simplify doing the above in the future, in case we need to, and it's not a big change on this PR neither

Reply:
Yeah. I went with `userConfiguration` way to be consistent; but I can see why the other choice is good.

Reply:
For consistency, do we want all the values in the `IndexedDBDataKeys` to end with `Data`?

Reply:
I dont think its a good idea to change previously set IndexedDB keys as this can lead to customer losing their assessment data.

Reply:
+1 to @Shobhit1 We actually cannot change this keys without having some sort of data migration thing going on... and that looks like out of the scope of this PR and feature.